### PR TITLE
Nightly : Fixed the URL to ping

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Nightly : Fixed the URL to ping
| Type?             | bug fix
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: 
| UI Tests          | N/A
| Fixed issue or discussion?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

**Review by boubkerbribri — DISMISSED** [review]

**Review by jolelievre — DISMISSED** [review]

File: .github/workflows/cron_nightly_tests_reusable.yml [hunk 1]
```diff
@@ -210,10 +210,14 @@ jobs:
         timeout-minutes: 15
         env:
           URL_FO: ${{ ((env.GH_BRANCH == '8.0.x') || (env.GH_BRANCH == '1.7.8.x')) && 'http://localhost:8001/' || 'https://localhost:8002/' }}
+          URL_PING: |
+            ${{ ((env.GH_BRANCH == '8.0.x') || (env.GH_BRANCH == '1.7.8.x'))
+            && (matrix.CAMPAIGN == 'sanity' && 'http://localhost:8001/install-dev/' || 'http://localhost:8001/en/')
+            || (matrix.CAMPAIGN == 'sanity' && 'https://localhost:8002/install-dev/' || 'https://localhost:8002/en/') }}
```
Comment (jolelievre) [inline .github/workflows/cron_nightly_tests_reusable.yml]:
```suggestion
          URL_PING: |
            ${{ matrix.CAMPAIGN == 'sanity' && '/install-dev/' || '/en/') }}
```

File: .github/workflows/cron_nightly_tests_reusable.yml [hunk 2]
```diff
@@ -210,10 +210,14 @@ jobs:
...
         run: |
           USER_ID=$(id -u) GROUP_ID=$(id -g) \
           docker-compose -f docker-compose.yml up -d --build
-          bash -c 'while [[ "$(curl -L -s -o /dev/null -w %{http_code} ${{ env.URL_FO }}en/)" != "200" ]]; do sleep 5; done'
+          bash -c 'while [[ "$(curl -L -s -o /dev/null -w %{http_code} ${{ env.URL_PING }})" != "200" ]]; do sleep 5; done'
```
Comment (jolelievre) [inline .github/workflows/cron_nightly_tests_reusable.yml]:
```suggestion
          bash -c 'while [[ "$(curl -L -s -o /dev/null -w %{http_code} ${{ env.URL_FO }}${{ env.URL_PING }})" != "200" ]]; do sleep 5; done'
```

**Review by boubkerbribri — DISMISSED** [review]

File: .github/workflows/cron_nightly_tests_reusable.yml (hunk 1 above)
Comment (Progi1984) [reply .github/workflows/cron_nightly_tests_reusable.yml]: Fixed

File: .github/workflows/cron_nightly_tests_reusable.yml (hunk 2 above)
Comment (Progi1984) [reply .github/workflows/cron_nightly_tests_reusable.yml]: Fixed

**Review by boubkerbribri — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @Progi1984

**Pull request merged**